d2bs/kolbot/libs/common/Pather.js

```diff
@@ -1,1696 +1,1701 @@
 /**
-*	@filename	Pather.js
-*	@author		kolton
-*	@desc		handle player movement
-*/
+ *    @filename    Pather.js
```

Comment:
This file needs to be resubmitted without all the spacing differences.

Reply:
Some story as Misc, do you have a git related fix for this? To remove these and the previous alternation (commits) to both Pather and Misc completely from my forked version?